discovery/gossiper.go

```diff
@@ -1157,16 +1172,21 @@ func (d *AuthenticatedGossiper) isRecentlyRejectedMsg(msg lnwire.Message) bool {
 
 // retransmitStaleChannels examines all outgoing channels that the source node
 // is known to maintain to check to see if any of them are "stale". A channel
-// is stale iff, the last timestamp of its rebroadcast is older then
-// broadcastInterval.
-func (d *AuthenticatedGossiper) retransmitStaleChannels() error {
+// is stale iff, the last timestamp of its rebroadcast is older than the
+// RebroadcastInterval. We also check if a refreshed node announcement should be
+// resent.
+func (d *AuthenticatedGossiper) retransmitStaleChannels(now time.Time) error {
```

Comment:
Rename the method to `retransmitStaleAnns` since we also do our node announcement now?

Reply:
Done: https://github.com/lightningnetwork/lnd/pull/2084/commits/ec91c340243da0e61b58a01aee5e5cebd411f5e1